test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+PGBACKREST_TEST_DIR=/home/gpadmin/test_pgbackrest
+PGBACKREST_BIN=/usr/local/bin
+GPHOME=/usr/local/greenplum-db-devel
+
+echo "Starting up demo cluster" > /dev/null
+source $GPHOME/greenplum_path.sh
+pushd gpdb_src/gpAux/gpdemo
+make create-demo-cluster WITH_MIRRORS=true
+source gpdemo-env.sh
+popd
+
+
+echo "Creating backup and log directories for pgbackrest..." > /dev/null
+TEST_NAME=$(basename "${0%.sh}")
+mkdir -p "$PGBACKREST_TEST_DIR/logs/$TEST_NAME"
+mkdir -p "$PGBACKREST_TEST_DIR/$TEST_NAME"
+
+DATADIR="${MASTER_DATA_DIRECTORY%*/*/*}"
+MASTER=${DATADIR}/qddir/demoDataDir-1
+PRIMARY1=${DATADIR}/dbfast1/demoDataDir0
+PRIMARY2=${DATADIR}/dbfast2/demoDataDir1
+PRIMARY3=${DATADIR}/dbfast3/demoDataDir2
+MIRROR1=${DATADIR}/dbfast_mirror1/demoDataDir0
+MIRROR2=${DATADIR}/dbfast_mirror2/demoDataDir1
+MIRROR3=${DATADIR}/dbfast_mirror3/demoDataDir2
+MASTER_PORT=6000
+PRIMARY1_PORT=6002
+PRIMARY2_PORT=6003
+PRIMARY3_PORT=6004
+
+echo "Filling the pgbackrest.conf configuration file" > /dev/null
+cat <<EOF > $PGBACKREST_TEST_DIR/pgbackrest.conf
+[seg-1]
+pg1-path=$MASTER
+pg1-port=$MASTER_PORT
+
+[seg0]
+pg1-path=$PRIMARY1
+pg1-port=$PRIMARY1_PORT
+
+[seg1]
+pg1-path=$PRIMARY2
+pg1-port=$PRIMARY2_PORT
+
+[seg2]
+pg1-path=$PRIMARY3
+pg1-port=$PRIMARY3_PORT
+
+[global]
+repo1-path=$PGBACKREST_TEST_DIR/$TEST_NAME
+log-path=$PGBACKREST_TEST_DIR/logs/$TEST_NAME
+start-fast=y
+fork=GPDB
+EOF
+
+echo "Initiallizing pgbackrest for GPDB" > /dev/null
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
+    --config $PGBACKREST_TEST_DIR/pgbackrest.conf --stanza=seg$i stanza-create
+done
+
+echo "Configuring WAL archiving command" > /dev/null
+gpconfig -c archive_mode -v on
+
+gpconfig -c archive_command -v "'PGOPTIONS=\"-c gp_session_role=utility\" \
+$PGBACKREST_BIN/pgbackrest --config $PGBACKREST_TEST_DIR/pgbackrest.conf \
+--stanza=seg%c archive-push %p'" --skipvalidation
+
+gpstop -ar
+
+echo "pgbackrest health check" > /dev/null
+for i in -1 0 1 2
+do 
+	PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
```

Comment:
replace the tab with spaces
```suggestion
  PGOPTIONS="-c gp_session_role=utility" $PGBACKREST_BIN/pgbackrest \
```